pkg/cloudprovider/aws/instance.go

```diff
@@ -249,17 +249,19 @@ func (p *InstanceProvider) instanceToNode(ctx context.Context, instance *ec2.Ins
 			if injection.GetOptions(ctx).GetAWSNodeNameConvention() == options.ResourceName {
 				nodeName = aws.StringValue(instance.InstanceId)
 			}
+			zone := aws.StringValue(instance.Placement.AvailabilityZone)
 			return &v1.Node{
 				ObjectMeta: metav1.ObjectMeta{
 					Name: nodeName,
 					Labels: map[string]string{
-						v1.LabelTopologyZone:       aws.StringValue(instance.Placement.AvailabilityZone),
+						v1.LabelTopologyZone:       zone,
+						v1.LabelTopologyRegion:     zone[:len(zone)-1],
```

Comment:
Does `zone[:len(zone)-1]` work for outposts / localzones etc?  Should we get region from an env var instead - I'm guessing an env var is set since the AWS SDK is sending requests to the right region

Reply:
This complexity is exactly the reason I didn't want to Karpenter to be region aware.